File: src/add-charges.ts
// Purchase cost in Euros per kWh, including VAT.
const purchaseCost = 0.0212;

// Energy tax in Euros per kWh, including VAT.
const energyTax = 0.15;

export default function addCharges(price: number): number {
  return price + purchaseCost + energyTax;
}

File: internal/add_charges.go
package internal

// Returns the price in Euros per kWh, including VAT.
func AddCharges(price float64) float64 {
	const (
		purchaseCost = 0.0484
		energyTax = 0.1312
	)

	return price + purchaseCost + energyTax
}
